core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -434,10 +436,15 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
     dataSource.inTransaction { dataAccess =>
       withWorkspaceContext(workspaceName, dataAccess) { workspaceContext =>
         requireAccessIgnoreLock(workspaceContext.workspace, WorkspaceAccessLevels.Owner, dataAccess) {
-          dataAccess.workspaceQuery.listEmailsAndAccessLevel(workspaceContext).map { emailsAndAccess =>
-            // toMap below will drop duplicate keys, keeping the last entry only
-            // sort by access level to make sure higher access levels remain in the resulting map
-            RequestComplete(StatusCodes.OK, emailsAndAccess.sortBy { case (_, accessLevel) => accessLevel }.toMap)
+          dataAccess.workspaceQuery.listEmailsAndAccessLevel(workspaceContext).flatMap { emailsAndAccess =>
+            dataAccess.workspaceQuery.getInvites(workspaceContext.workspaceId).map { invites =>
+              // toMap below will drop duplicate keys, keeping the last entry only
+              // sort by access level to make sure higher access levels remain in the resulting map
+              val granted = emailsAndAccess.sortBy { case (_, accessLevel) => accessLevel }.map { case (email, accessLevel) => email -> AccessEntry(accessLevel, false)}
+              val pending = invites.sortBy { case (_, accessLevel) => accessLevel }.map { case (email, accessLevel) => email -> AccessEntry(accessLevel, true)}
+
+              RequestComplete(StatusCodes.OK, WorkspaceACL((granted ++ pending).toMap))
```

Comment:
This looks like a change in return type without a corresponding change in rawls.yaml. However, it's not obvious to me that rawls.yaml is incorrect. I'm not positive that anything needs to be done there, but I think it's worth a double-check.